charts/apiclarity/templates/clusterrole.yaml

```diff
@@ -7,8 +7,14 @@ metadata:
     {{ include "apiclarity.labels" . }}
 rules:
 - apiGroups: [""]
-  resources: ["nodes", "services", "pods"]
+  resources: ["nodes", "services", "pods", "configmaps"]
+  verbs: ["get", "list", "watch", "create"]
+- apiGroups: [""]
+  resources: ["configmaps"]
   verbs: ["get", "list", "watch"]
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["create", "delete"]
```

Comment:
better to put it in a role and not clusterrole since we don't need permissions to delete/create secrets in all namespaces, right?

There is already `charts/apiclarity/templates/role.yaml` and `charts/apiclarity/templates/rolebinding.yaml` just need to update the comment and remove the `{{- if .Values.global.traceSampling.enable -}}` & `{{- end -}}` from both files

Reply:
I have already tried to remove my stuff from ```clusterrole.yaml``` to use ```role.yaml``` by it was not a success. Note that I hadn't used ```rolebinding.yaml```.
I will try again

Reply:
Ok, with the update in ```rolebinding.yaml``` it works. I revert clusterrole.yaml to the original state.